test/rules/assertions/requireParam.js

```diff
@@ -414,19 +480,27 @@ export default {
       /**
        *
        */
-      function quux ({bar, baz}, foo) {
+      function quux ({bar, baz___}, foo) {
```

Comment:
Also here...

Reply:
This was accidentally leftover from debugging, so I've removed it.